### PR TITLE
[Chart] Add LineChartView

### PR DESCRIPTION
**Pull request by jkpu**
### Description of Change ###
Add LineChartView (Single and Multi Line chart )

### Bugs Fixed ###
N/A

### API Changes ###
Added:
- Add LineChartView 
- Add TestCase of LineChartView

### Behavioral Changes ###
LineChartView screenshot is following images.

LineChart1

LineChart2

MultiLineChart

### CODE REVIEW

File: src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+    /// <summary>
+    /// LineDataItemGroup is an inheritance type of DataItemGroup for the LineChartView.
+    /// </summary>
+    public class LineDataItemGroup : DataItemGroup
+    {
+        Color _fgColor;
+        byte _fgColorAlpha;
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs:28]: You can remove all constructor of `LineDataItemGroup` if change like below
```suggestion
        Color _fgColor = Color.Transparent;
        byte _fgColorAlpha = 0xFF;
```

File: src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                _fgColor = value;
+                OnPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets a transparency value of line foreground color.
+        /// An alpha value of 0 is fully transparent and an alpha value of 0xFF is fully opaque.
+        /// </summary>
+        public byte ForegroundColorAlpha
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs:69]: Why need it?
Without it, We can set transparency with ForegroundColor

**Comment by myroot** [comment]
All properties are not updated on Canvas until call `UpdateChart` method, but it is make hard to use on XAML

File: src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/LineChartViewRenderer.cs [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+                    {
+                        _dataTable[i, j] = items[j] as IDataItem;
+                    }
+                }
+            }
+        }
+
+        protected void CalculateAxisSize()
+        {
+            if (Element.AxisOption != null)
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/LineChartViewRenderer.cs:207]: `Element.AxisOption` can't be null

File: src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs (hunk 1 above)
Comment (jkpu) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: Done. but remain constructor preventing build error when user make new LineDataItemGroup with parameters.

File: src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs (hunk 2 above)
Comment (jkpu) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: Done. remove this property.

File: src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/LineChartViewRenderer.cs (hunk 3 above)
Comment (jkpu) [reply src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/LineChartViewRenderer.cs]: Done. removed this code.

File: src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs (hunk 1 above)
Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: Could you explain more details?
I don't understand which error was occurred?

Comment (jkpu) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: After I removed all  constructor in `LineDataItemsGroup`, I got the following error.
"LineDataItemGroup has no constructor using 2 parameter "

```c#
var lineDataItemGroup = new LineDataItemGroup(colorDummyDataItems, "Weekly data with each color and forground color");
```

Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: hm.. base class has a constructor with two parameter, so i think, something wrong..

Comment (jkpu) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1729

LineDataItemGroup class has only a parameterless constructor. it make cs1729 error when build the following code.

```c#
var lineDataItemGroup = new LineDataItemGroup(colorDummyDataItems, "Weekly data with each color and forground color");
```

Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs]: Yes, you are right, my mistake

**Pull request merged**